support/c/idris_file.c

```diff
@@ -15,6 +15,10 @@
 #include <sys/select.h>
```

Comment:
It seems (from reading some manpages), that `<sys/wait.h>` can (and should) be included on all OSs (except Windows). So maybe the include would be a better fit up here?

```suggestion
#include <sys/select.h>
#include <sys/wait.h>
```